config/webpack/browser.config.babel.js

```diff
@@ -6,6 +6,20 @@ import { WebpackBundleSizeAnalyzerPlugin } from 'webpack-bundle-size-analyzer';
 import LodashModuleReplacementPlugin from 'lodash-webpack-plugin';
 import TerserPlugin from 'terser-webpack-plugin';
 
+const module = {
+  rules: [
+    {
+      test: /\.js$/,
+      // `cross-fetch` requires a number of polyfills (like Promise), so we
+      // let `@babel/plugin-transform-runtime` inject them.
+      exclude: /node_modules\/(?!cross-fetch\/)/,
+      use: {
+        loader: 'babel-loader',
```

Comment:
Are we sure that by removing `babelrc: true`, the webpack is using the `babel.config.json`?

Reply:
`babelrc` defaults to `true` when a filename is set: https://babeljs.io/docs/en/options#babelrc
`babel-loader` sets the `filename`: https://github.com/babel/babel-loader/blob/master/src/index.js#L131

However, I think the `babelrc` option is only relevant for `.babelrc` files, hence why it requires a filename (as `.babelrc` options are merged on top of one another based on where each file is in the project tree. This does not apply to `babel.config.js` files, which just apply over the entire project and even `node_modules`).

Reply:
Thanks for explanation.